package.json

```diff
@@ -4,13 +4,15 @@
   "description": "Minify JavaScript files with UglifyJS.",
   "main": "index",
   "scripts": {
-    "test": "gulp test"
+    "test": "nyc mocha",
```

Comment:
`mocha test/index.js` is safer, even though it's unlikely there'll be more than one test files but who knows. `nyc` is only applicable for coverage.

Reply:
fixed